homestyle.css

```diff
@@ -187,13 +174,26 @@ a:active{
 	width: 100%;
 	height: 100%;
 	padding: 10%;
+	border: #000000 solid 2px;
```

Comment:
Convention would put this as
```
border: 2px solid #000;
```
i.e. your ordering is unusual. Same for your other borders in your CSS.